romtest/test_rom.py

```diff
@@ -0,0 +1,225 @@
+import argparse
+import pathlib
+import subprocess
+import sys
+import typing
+from dataclasses import dataclass
+from pathlib import Path
+
+THIS_FILE = pathlib.Path(__file__).absolute().parent
+
+NES_SCRREN_ROWS = 30
+NES_SCRREN_COLS = 32
+
+
+@dataclass
+class Test:
+    rom: str
+    ascii: str
```

Comment:
pass_pattern or something maybe? I had to look at how this was used to understand what it was. :P